ert_shared/dark_storage/endpoints/experiments.py

```diff
@@ -1,25 +1,39 @@
-from uuid import UUID
+import os
+from uuid import UUID, uuid4
 from typing import Any, Mapping, List
 
 from fastapi import APIRouter, Body, Depends
 from ert_storage import json_schema as js
 
-from ert_shared.dark_storage.enkf import LibresFacade, get_res
-
+from ert_shared.dark_storage.enkf import LibresFacade, get_res, get_id
 
 router = APIRouter(tags=["experiment"])
 
 
 @router.get("/experiments", response_model=List[js.ExperimentOut])
 def get_experiments(*, res: LibresFacade = Depends(get_res)) -> List[js.ExperimentOut]:
-    raise NotImplementedError
+    return [
+        js.ExperimentOut(
+            name="default",
```

Comment:
This name could be something else right? Should we have it hardcoded for the time being or should me make it work without hardcoding?